drud/ddev
# Add context to internet-connection-not-detected and add docs, fixes #2409

## The Problem/Issue/Bug:

People have started seeing "Internet connection not detected", but that message doesn't give them any context about what it means or why.

## How this PR Solves The Problem:

* Add `ddev config global --internet-detection-timeout-ms`
* Add link to docs in "Internet connection not detected" message
* Add docs in FAQ about what to do.
* Add more in the config-yaml section.

## Manual Testing Instructions:

* Make the problem happen with a 1000ms delay in Parallels or any network conditioner
* See the output with the link. (The link won't work until release time)
* Fix with `ddev config global --internet-detection-timeout-ms=3000`

## Related Issue Link(s):

* #2409 immediately prompted this (getting message due to slow internet)
* PR #2112  fixed #1969 by adding the timeout
* #2302 (especially https://github.com/drud/ddev/issues/2302#issuecomment-641323539) gives background debugging information and how to find out how to use dig to check dns lookup.
